src/Components/Login.css

```diff
@@ -0,0 +1,17 @@
+body {
+   background-image: url("/Wittenberg_crop.jpeg");
+   background-repeat: no-repeat;
+   background-position: center;
+   background-size: cover;
+   background-color: #cccccc; 
+   height: 100vh;
+ }
+
+ #hero-text {
```

Comment:
If hero-text is only occurring in one place, then referencing the id is a good option. Otherwise, consider utilizing a class instead. The styles look good though!